src/D2L.Bmx/Write.cs

```diff
@@ -0,0 +1,87 @@
+
+public class Write {
+	public static void WriteHandler( string? org, string? user, string? account, string? role, int? duration, bool nomask, string? output, string? profile ) {
+		// TODO: Get values of org, user, account, role and profile from the config file and assign them locally. This way we can check if
+		// values are null to see if they have been set through the commandline or config file
+
+		var unsetFlags = new List<string>();
+
+		// ask user to input org if org flag isn't set
+		if( string.IsNullOrEmpty( org ) ) {
+			unsetFlags.Add( "\"org\"" );
+		};
+
+		// check if profile flag has been set
+		if( string.IsNullOrEmpty( profile ) ) {
+			unsetFlags.Add( "\"profile\"" );
+		}
+
+		if( unsetFlags.Count != 0 ) {
+			Console.Write( "Error: required flag(s) " );
```

Comment:
I assume this is the current behaviour of the Go version? If so, this is fine. But it does seem unintuitive to me that missing some flags results in a prompt for input, whereas others result in error exit. I wonder if we can/should change that in the future.